rootless-install.sh

```diff
@@ -25,6 +25,11 @@ init_vars() {
 	if systemctl --user daemon-reload >/dev/null 2>&1; then
 		SYSTEMD=1
 	fi
+
+	USE_SYSTEM_DOCKER=
+	if command -v docker >/dev/null 2>&1; then
+		USE_SYSTEM_DOCKER=1
```

Comment:
This should not be set by default because of potential version mismatch

Reply:
Better in 1bb6e67?